docker-compose.yml

```diff
@@ -334,23 +247,35 @@ services:
-      /entrypoint.sh call --json_input --json_output localhost:50151 CreateBridgePort    "{\"bridge_port\" : {\"spec\" : {\"mac_address\" : \"qrvMAAAB\", \"ptype\": \"TRUNK\" , \"logical_bridges\": [] }}, \"bridge_port_id\" : \"eth3\" }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateVrf            "{\"vrf_id\"            : \"blue\",          \"vrf\" :             {\"spec\" : {               \"loopback_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772418}, \"len\": 24}, \"vtep_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772162}, \"len\": 24} }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateVrf            "{\"vrf_id\"            : \"green\",         \"vrf\" :             {\"spec\" : {\"vni\" : 100, \"loopback_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772674}, \"len\": 24}, \"vtep_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772162}, \"len\": 24} }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateVrf            "{\"vrf_id\"            : \"yellow\",        \"vrf\" :             {\"spec\" : {\"vni\" : 101, \"loopback_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772930}, \"len\": 24}, \"vtep_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772162}, \"len\": 24} }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vlan10\",        \"logical_bridge\" :  {\"spec\" : {\"vni\" : 10, \"vlan_id\": 10 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vlan20\",        \"logical_bridge\" :  {\"spec\" : {\"vni\" : 20, \"vlan_id\": 20 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vlan30\",        \"logical_bridge\" :  {\"spec\" : {\"vni\" : 30, \"vlan_id\": 30 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vlan40\",        \"logical_bridge\" :  {\"spec\" : {\"vni\" : 40, \"vlan_id\": 40 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vlan50\",        \"logical_bridge\" :  {\"spec\" : {              \"vlan_id\": 50 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateSvi            "{\"svi_id\"            : \"blue-vlan20\",   \"svi\" :             {\"spec\" : {\"vrf\": \"//network.opiproject.org/vrfs/blue\",   \"logical_bridge\": \"//network.opiproject.org/bridges/vlan20\", \"mac_address\" : \"qrvMAAAh\", \"gw_ip_prefix\": [{\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 336860161}, \"len\": 24}] }} }" && \
```

Comment:
Can we use vlan20 instead of blue-vlan20 for SVI name? and logical_bridge_id as br<vlan-id>? ,e.g br50

Reply:
I specifically want the SVI name to reflect the binding between VRF and logical bridge.
And I don't see any reason to encode the object type as prefix into the logical bridge objects. We are not doing that for other object types either. How we call the underlying Linux realization entities is a different matter.

Reply:
How we call the underlying Linux realization entities is a different matter.
-> This is the problem, unnecessary lengthy names lead to have additional mapping in the backend as Linux interface names length restricted to 15 characters. IMO, it is easy to get and know the SVI to VRF mappings, if we want to rebind the SVI from one VRF to another VRF on the fly, I would need to delete the SVI and create with new VRF name part of SVI name.

Reply:
I agree with @venkatmahalingam on this problem
```
if we want to rebind the SVI from one VRF to another VRF on the fly, I would need to delete the SVI and create it with new VRF name part of SVI name.
```
But naming of objects is left to the user, there is no code or rules to mandate to use any names... so we as OPI should not care... 
user will pick a name or  (since resource ID) is optional, grpc server will generate UUID as a name (or more accurately resource ID) instead...

Reply:
How realistic and important a use case is it to move a subnet on a logical bridge to another VRF without any traffic impact? To me that is a change in network planning, which typically has wider implications in the network.
I know it is possible in Linux to shift an existing SVI netdev to another VRF just by enslaving it to the new vrf netdev, but is this something the EVPN-GW API must cover? In fact, I think it currently does cover it, as the VRF reference in the SVI spec is not marked as `immutable`. Same goes for the LogicalBridge reference, by the way, but the VLAN ID of an SVI netdev can't be changed in Linux on the fly. 
At any rate, the object naming arbitrary, and we can apply any convention we like in this example configuration.